lib/eventFactory.js

```diff
@@ -15,20 +15,28 @@ let instance;
  * @param  {Object}   config.pipelineConfig
  * @param  {Object}   config.pipelineConfig.workflowGraph   Object with nodes and edges that represent the order of jobs
  * @param  {String}   config.startFrom                      Startfrom (e.g. ~commit, ~pr, ~sd@123:main, etc)
+ * @param  {String}   config.branch                         triggered branch name
```

Comment:
can you sort this comment alphabetically?

Reply:
I will fix!

Reply:
I also fixed this comment.
https://github.com/screwdriver-cd/models/blob/master/lib/eventFactory.js#L37